bitbucket/resource_branching_model.go

```diff
@@ -302,11 +313,6 @@ func flattenBranchModel(rp *BranchModel, typ string) []interface{} {
 		"name":                  rp.Name,
 	}
 
-	// if production branch is disabled it wont show up in response and will show up without the proerty if enabled
```

Comment:
This workaround should no longer be needed now that we're using the `settings` URL.